lisa_commands/lambda_function.py

```diff
@@ -39,3 +45,29 @@ def lambda_handler(data, _context):
 
     request.submit_slack_request(data, chat_action)
     return request.return_status()
+
+def valid_slack_request(data):
+    """
+    Did this request come from slack
+    data - the data object (dict)
+    """
+    slack_signature = data['headers']['X-Slack-Signature']
```

Comment:
Is there any chance these dict lookups could fail? If so, using dict.get() is a safer way, as it will return None if something isn't found (instead of raising a KeyError).

Reply:
If it's a valid slack request those should always be there. But I'll go ahead and toss
the functions in there for knowledge sake